charts/argocd-progressive-rollout/crds/crd-progressiverollout.yaml

```diff
@@ -0,0 +1,230 @@
+---
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
This is just a copy from /config right? I do not know what the best practice is, but can we reference the file from there directly instead so that we do not have to worry about them going out of sync? Or could the helm charts be stored inside of config/crd without messing up with kustomize?

Reply:
It's a copy&paste. We can't reference the file because in Helm they need to be YAML file as per https://helm.sh/docs/chart_best_practices/custom_resource_definitions/

It's best practice to keep them separate, so we can't add the helm chart inside `config/crd`. I think it would confuse people having an Helm chart stored under a Kustomize subdirectory. What do you think?

Reply:
I know that we said that we do not expect the crd definition to change often, so I guess it is fine to keep it as a copy for now. I agree that it would be awkward to have a chart inside of a kustomize subdir. Maybe in the future we can extend `make generate` or some other make target just to make sure that these are kept in sync?

Reply:
this is a good idea.

Reply:
or a symlink?

Reply:
Yeah, that's a good idea.